Developers/Writing_Good_Commits.rst

```diff
@@ -8,13 +8,13 @@ At coala we are looking heavily at the maintainability of the code.
     Code is more often read than written!
 
 It is obvious that we need good code. In order to do that we are
-verifying that every change to our code (i.e. the commits) is making it
+verifying that every change in our code (i.e. the commits) is making it
```

Comment:
Umm is this change valid? You "make changes *to* the code*. But I guess `in` sounds alright too. Not sure.